package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "bodymovin",
-  "version": "4.6.7",
-  "description": "After Effects plugin for exporting animations to SVG + JavaScript or canvas + JavaScript",
+  "version": "4.6.9",
+  "description": "Adam Chen Wei new - After Effects plugin for exporting animations to SVG + JavaScript or canvas + JavaScript",
```

Comment:
@goel4ever can't squash, I can remove the description, that was for testing, my fault.